### PR TITLE
feat(GiniHealthSDK): Install App Bottom Sheet IPC-233

### PR DESCRIPTION
**Pull request by razvancapra**
https://ginis.atlassian.net/browse/IPC-233

### CODE REVIEW

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+import UIKit
+import GiniHealthAPILibrary
+
+protocol InstallAppBottomViewProtocol: AnyObject {
+    func didTapOnContinue()
+}
+
+final class InstallAppBottomViewModel {
+    
+    var giniHealthConfiguration = GiniHealthConfiguration.shared
```
Comment (zladzeyka) [inline HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift:17]: [Suggestion]: what about to use here only a button configuration and don't expose `GiniHealthConfiguration` from the model?

Comment (razvancapra) [reply HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift:17]: We need these expose here for the `titleLabelFont` and for `moreInformationLabelFont` fonts required for UI

**Comment by razvancapra** [comment]
> Great Job @razvancapra 👏🏻 I've spotted on the SE device a small layout issue: the bottom sheet gets too long :)
> 
> Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-30 at 17 49 09 Simulator Screenshot - iPhone SE (3rd generation) - 2024-04-30 at 17 49 02

Yes. I will fix this in the next PR that contains OpenWith Onboarding bottom sheet where I introduced the new dynamic bottom sheet that includes minimium height as well.

File: HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift (hunk 1 above)
Comment (zladzeyka) [reply HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/InstallAppBottomViewModel.swift:17]: I guess it might me `let` then :)

**Review by zladzeyka — APPROVED** [review]
Thank you @razvancapra

**Pull request merged**